### PR TITLE
Add metacello installation instruction to readme

### LINKED ISSUE #103: Metacello installation
I would like to have installation instructions for Metacello

### PR DESCRIPTION
**Pull request by gritfessel**
Closes #103

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,5 +14,22 @@ In order to use our tool, you have to install it in your Squeak image. Our prefe
 - For later updates, you can just pull the new version from the Git Repository. To do that, select the `dev` branch and click the button <kbd>Pull</kbd>.
 - Once the new version was downloaded, right click the latest commit and choose <kbd>Checkout objects</kbd>.
 
+### Metacello
+- Open a workspace and execute
+```
```
Comment (marcelgarus) [inline README.md:19]: Let's add Smalltalk syntax highlighting!
```suggestion
```smalltalk
```

File: README.md [hunk 2]
```diff
@@ -14,5 +14,22 @@ In order to use our tool, you have to install it in your Squeak image. Our prefe
...
+Metacello new
+    baseline: 'ChangesReloaded';
+    repository: 'github://hpi-swa-teaching/ChangesBrowser:dev/packages';
+    load.
```
Comment (marcelgarus) [inline README.md:23]: No dot
```suggestion
    load
```

File: README.md [hunk 3]
```diff
@@ -14,5 +14,22 @@ In order to use our tool, you have to install it in your Squeak image. Our prefe
...
+```
+as well as
+```
```
Comment (marcelgarus) [inline README.md:26]:
```suggestion
```smalltalk
```

File: README.md [hunk 4]
```diff
@@ -14,5 +14,22 @@ In order to use our tool, you have to install it in your Squeak image. Our prefe
...
+Metacello new
+    baseline: 'Changes';
+    repository: 'github://hpi-swa-teaching/ChangesBrowser:dev/packages';
+    load.
```
Comment (marcelgarus) [inline README.md:30]:
```suggestion
    load
```

File: README.md [hunk 5]
```diff
@@ -14,5 +14,22 @@ In order to use our tool, you have to install it in your Squeak image. Our prefe
...
 
+### Metacello
+- Open a workspace and execute
+```smalltalk
+Metacello new
+    baseline: 'ChangesReloaded';
+    repository: 'github://hpi-swa-teaching/ChangesBrowser:dev/packages';
+    load
+```
+as well as
```
Comment (SilvanVerhoeven) [inline README.md:25]:
```suggestion
and after that
```

Comment (gritfessel) [reply README.md:25]: The order actually doesn't matter, so I would leave it the way it is...

**Pull request merged**